dev-docs/source/Testing/SliceViewer/SliceViewer.rst

```diff
@@ -40,92 +41,104 @@ Data
 	SXD23767 = Load(Filename='SXD23767.raw', LoadMonitors='Exclude')
 	# Set some UB with angles we can play with
 	SetUB(SXD23767, 1,1,2,90,90,120)
-	md_non_ortho = ConvertToDiffractionMDWorkspace(InputWorkspace='SXD23767', OutputDimensions='HKL')
+	SXD_MD_nonortho = ConvertToDiffractionMDWorkspace(InputWorkspace='SXD23767', OutputDimensions='HKL')
```

Comment:
Good idea to rename the variable here - makes it easier to understand which workspace is related to which. However will need to change the text in Test 10 as this refers to md_non_ortho.

Reply:
> * Were you going to change the URL in the Introduction as part of this PR?

Yes, thanks for reminding me!

> * Test section 7 (line subplots and roi). The text here seems to suggest I shouldn't be able to use this functionality with `SXD_MD_nonortho` . I have tried with this file and this functionality is available so I'm a little confused. Has the functionality changed since this was written or have I misunderstood which files it shouldn't work with?

The functionality is only disabled when non-orthogonal view is turned on - I will amend this to clarify.